### PR TITLE
✨(Feature): Created Job Summary Pages 

### PR DESCRIPTION
**Pull request by PxttytheThxtty**

### CODE REVIEW

File: libs/au-pair-dashboard/feature/src/lib/au-pair-dashboard.component.ts [hunk 1]
```diff
@@ -232,6 +232,16 @@ export class AuPairDashboardComponent implements OnInit {
     }
   }
 
+  async checkHasEmployerSummary(){
```
Comment (Ruben-Brits) [inline libs/au-pair-dashboard/feature/src/lib/au-pair-dashboard.component.ts:235]: viewChild imported but never used

File: libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.scss [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    background-color: var(--ion-color-primary);
+    margin: auto;
+    display: block;
+    margin-bottom: 30px;
+}
+
+.reject-button{
+    padding: 10px 22px;
+    gap: 10px;
+    border: 2px solid var(--ion-background-color);
```
Comment (ricecakesreece) [inline libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.scss:163]: Is this border meant to blend in with the background? And can we maybe look at changing the reject button colour from danger just so its not necessarily "bad" to reject. Maybe we make it grey

File: libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      providers: [API,ModalController]
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(JobSummaryModalComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
```
Comment (Ruben-Brits) [inline libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.spec.ts:30]: Are we going to add tests later or can this component's functions not be tested really?

File: libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+import { HttpClientTestingModule } from "@angular/common/http/testing";
+import { NavbarModule } from '@the-au-pair/shared/components/navbar';
+import { CommonModule } from '@angular/common';
+import { RouterTestingModule} from '@angular/router/testing';
+import { API } from '../../../../shared/api/api.service';
+import { By } from '@angular/platform-browser';
```
Comment (Ruben-Brits) [inline libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.spec.ts:9]: By defined but not used

File: libs/job-summary-parent-view/feature/src/lib/job-summary-parent-view.component.ts [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+    const navigation = this.router.getCurrentNavigation();
+    if(navigation !== null)
+      if(navigation.extras !== null)
+      { 
+        this.auPairID = navigation.extras.state?.['id'];
+      }
+  }
+
+  async ngOnInit(): Promise<void> {
+    console.log("This au pair ID is: ", this.auPairID);
```
Comment (Ruben-Brits) [inline libs/job-summary-parent-view/feature/src/lib/job-summary-parent-view.component.ts:97]: think we can remove this console log unless still used for testing?

File: libs/job-summary-parent-view/feature/src/lib/job-summary-parent-view.component.ts [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+  }
+
+  getCurDay(days : string[]) : number {
+    const pipe = new DatePipe('en-US');
+    const dateStr = pipe.transform(Date.now(),'EEEE');
+    return days.findIndex(x => x === dateStr);
+  }
+
+  async getActivities(){
+    console.log("getActivities");
```
Comment (Ruben-Brits) [inline libs/job-summary-parent-view/feature/src/lib/job-summary-parent-view.component.ts:162]: console log

File: libs/shell/feature/src/lib/shell-routing.module.ts [hunk 7]
```diff
@@ -199,6 +199,16 @@ const routes: Routes = [
       ),
     canLoad: [AuthGuard],
   },
+  {
+    path: 'job-summary-parent-view',
```
Comment (Ruben-Brits) [inline libs/shell/feature/src/lib/shell-routing.module.ts:203]: Please add the auth guards so the routes arent accessible by non logged in users,

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]

File: libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.spec.ts [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  let fixture: ComponentFixture<JobSummaryModalComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [JobSummaryModalComponent],
+      imports: [IonicModule, CommonModule,HttpClientTestingModule,NavbarModule, RouterTestingModule, FormsModule, NgxsModule.forRoot([AppState])],
+      providers: [API,ModalController]
+    }).compileComponents();
+  });
+
```
Comment (ricecakesreece) [inline libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.spec.ts:24]: Is it possible to add more testing in this PR?

File: libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.scss [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+    background-color: #69A88DB2;
+    margin: auto;
+    display: block;
+    margin-top: 25px;
+}
+
+.hire-button{
+    padding: 10px 22px;
+    gap: 10px;
+    border: 2px solid var(--ion-color-secondary);
```
Comment (ricecakesreece) [inline libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.scss:165]: Double checking that the borders on buttons are meant to be this way. Ignore if they are

File: libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.scss [hunk 10]
```diff
@@ -0,0 +1,386 @@
...
+    margin-left: 5px;
+}
+  
+.content {
+    font-weight: 500;
+    font-size: 14px;
+    letter-spacing: 0.05px;
+    color: white;
+}
+
```
Comment (ricecakesreece) [inline libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.scss:386]: no new line at end

File: libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.spec.ts [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+  let fixture: ComponentFixture<JobSummaryAuPairViewComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [JobSummaryAuPairViewComponent],
+      imports: [IonicModule, CommonModule,HttpClientTestingModule,NavbarModule, RouterTestingModule, FormsModule, NgxsModule.forRoot([AppState])],
+      providers: [API]
+    }).compileComponents();
+  });
+
```
Comment (ricecakesreece) [inline libs/job-summary-au-pair-view/feature/src/lib/job-summary-au-pair-view.component.spec.ts:25]: If possible please add testing for all of these new pages in this PR

**Review by Ruben-Brits — APPROVED** [review]

**Review by d-azmanov — APPROVED** [review]

**Pull request merged**